internal/resources/provisioner/provisioner_mock_test.go

```diff
@@ -0,0 +1,166 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package provisioner
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/go-test/deep"
+	"github.com/jarcoal/httpmock"
+
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/helper"
+	objectmetamodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/objectmeta"
+	provisionermodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/provisioner"
+)
+
+func setupHTTPMocks(t *testing.T) {
+	httpmock.Activate()
+	t.Cleanup(httpmock.Deactivate)
+}
+
+const (
+	https        = "https:/"
```

Comment:
should be` https        = "https://"` ? instead of `https        = "https:/"` ?

Reply:
slash (/) is appened to https:/ in the function https://github.com/vmware/terraform-provider-tanzu-mission-control/blob/main/internal/helper/request_url.go#L16